README.md

```diff
@@ -1 +1,74 @@
-# Demineur
\ No newline at end of file
+# Demineur
+
+Ceci est un simple programme qui vise a recreer le celebre jeu de demineur, qui est un jeu de reflexion dont le but est de
+
+localiser des mines cachees dans un champ virtuel avec pour seule indication le nombre de mines dans les zones adjacentes.
+
+
+## Regles du jeu:
+
+Vous disposez d'une grille contenant des mines cachees. 
+
+En choisissant une case vous connaissez le nombre de mines se trouvant dans les cases qui l'entourent.
+
+Le but du jeu est de detecter toutes les mines sans tomber exactement dessus.
+
+Bonne chance !
+
+
+## Fonctions:
+
+*
+*
+*
+*
+*
```

Comment:
void Ini
void AfficheT
void placeM
void jeu
void Nb_Mines_Voisines

Reply:
fonctions ajouté

Reply:
void Initialisation(tab,tab);
void Affichage(tab);
void Mines_Rand(tab);
void jeu(tab,tab);
void Nb_Mines_Voisines(tab,tab,int,int);

Reply:
bien noté